pages/el/_sass.md

```diff
@@ -9,59 +9,46 @@
 
-Πέρα από τους preprocessors, πρέπει να επισημάνουμε και εργαλεία post-processing, τα οποία έχουν αποκτήσει σημαντική δημοσιότητα τους τελευταίους μήνες, κυρίως χάρη στο [PostCSS](https://github.com/postcss/postcss) και στο [cssnext](https://cssnext.github.io/). Συχνά αναφέρονται ως «postprocessors» επειδή κάνουν transpile επερχόμενες συντακτικές αλλαγές σε τωρινή CSS. Πέρα απ' αυτό, είναι πάνω κάτω ισοδύναμοι με τους preprocessors με την εξαίρεση ότι δεν παρέχουν τίποτα περισσότερο από επερχόμενες συντακτικές δυνατότητες της CSS.
+Πέρα από τους preprocessors, πρέπει επίσης να επισημάνουμε εργαλεία όπως το [PostCSS](https://github.com/postcss/postcss) και το [cssnext](https://cssnext.github.io/) τα οποία έχουν γίνει αρκετά δημοφιλή τους τελευταίους μήνες.
 
-Δες τους postprocessors σαν ένα polyfill για δυνατότητες της CSS που δεν υποστηρίζονται ακόμα. Για παράδειγμα, μπορείς να γράψεις μεταβλητές όπως περιγράφονται στις [προδιαγραφές της CSS](http://dev.w3.org/csswg/css-variables/), και ύστερα να κάνεις compile τα stylesheets με έναν postprocessor, με αποτέλεσμα όπου εμφανίζεται η μεταβλητή να έχει αντικατασταθεί από την τιμή της, όπως θα έκανε και η Sass.
+Το PostCSS συνηθίζεται να αποκαλείται (λανθασμένα) ως “postprocessor”. Η υπόθεση, συνδιασμένο με το ατυχές όνομα, είναι πως το PostCSS κάνει parse τη CSS η οποία έχει ήδη επεξεργαστεί από έναν preprocessor. Ένω μπορεί να δουλέψει και με αυτό το τρόπο, δεν είναι απαιτούμενο και έτσι το PostCSS είναι στην πραγματικότητα απλώς ένας “processor”.
 
-Το σκεπτικό πίσω από τους postprocessors είναι ότι μόλις οι φυλλομετρητές υποστηρίξουν νέες δυνατότητες (π.χ. μεταβλητές CSS), ο postprocessor δεν τις κάνει compile πια και αφήνει τους φυλλομετρητές να τις διαχειριστούν.
+ Η ιδέα πίσω απο τους postprocessors είναι πως μόλις οι browsers υποστηρίξουν νέα χαρακτηριστικά (λ.χ. CSS variables), ο postprocessor δεν τους κάνει πια compile και ετσι επιτρέπει τους browsers να αναλάβουν.
 
-Παρότι η ιδέα του να παρέχουμε υποστήριξη για μελλοντικές συντάξεις ακούγεται αξιότιμη ιδέα, μπορώ να πως πως προτιμώ να χρησιμοποιώ Sass για τις περισσότερες εργασίες. Πάραυτα, υπάρχουν μερικές περιπτώσεις που πιστεύω ότι οι postprocessors είναι καταλληλότεροι από τη Sass και τα συναφή - το CSS prefixing για παράδειγμα - αλλά θα επανέλθουμε πάνω σ'αυτό.
+Παρέχει την πρόσβαση σε “tokens” των stylesheets σας (όπως τους selectors, τα properties και τις τιμές), τα επεξεργάζετε μέσω JavaScript για να εκτελέσει κάποια εργασία και τέλος να κάνει compile τα αποτελέσματα σε CSS. Για παράδειγμα, τη δημοφιλή prefixing βιβλιοθήκη [Autoprefixer](https://github.com/postcss/autoprefixer) φτιαγμένη πάνω στο PostCSS. Κάνει parse κάθε CSS rule για να δεί αν χρειάζονται vendor prefixes βασισμένο στο εργαλείο by συσχέτισης υποστήριξης χαρακτηριστικών στους browsers [CanIUse](http://caniuse.com) και μετά διαγράφει vendor prefixes που δεν χρειάζονται και προσθέτει αυτά που χρειάζονται.
```

Comment:
Hey man thanks for the quick reply, I will take a look at it in a few hours and get back to you